po/it.po

```diff
@@ -0,0 +1,703 @@
+
+msgid "Compute SHA512 hash sum"
+msgstr "Calcola il codice hash SHA512"
+
+msgid "MD5 Checksum"
+msgstr "Codice di controllo MD5"
+
+msgid "Compute MD5 hash sum"
+msgstr "Calcola il codice hash MD5"
+
+msgid "Base64 Encode"
+msgstr "Codifica in Base64"
+
+msgid "Encode text to Base64"
+msgstr "Codifica il testo in Base64"
+
+msgid "Base64 Decode"
+msgstr "Decodifica da Base64"
+
+msgid "Decode text from Base64"
+msgstr "Decodifica il testo da Base64"
+
+msgid "Remove Trailing"
+msgstr "Rimuovi spazi bianchi in eccesso"
+
+msgid "Remove trailing whitespaces and newlines"
+msgstr "Rimuove spazi a fine riga e righe vuote a fine testo"
+
+msgid "Trim Lines"
+msgstr "Snellisci le righe"
+
+msgid "Remove leading and trailing whitespaces and newlines"
+msgstr ""
+"Rimuove spazi a fine o inizio riga e righe vuote a fine o inizio testo"
+
+msgid "Count Symbols"
+msgstr "Conta simboli"
+
+msgid "Count how many symbols are in text"
+msgstr "Conta il numero di simboli nel testo"
+
+msgid "Count Lines"
+msgstr "Conta righe"
+
+msgid "Count how many lines are in text"
+msgstr "Conta il numero di righe nel testo"
+
+msgid "Count Words"
+msgstr "Conta parole"
+
+msgid "Count how many words are in text"
+msgstr "Conta il numero di parole nel testo"
+
+msgid "Prettify JSON"
+msgstr "Abbellisci JSON"
+
+msgid "Parse JSON and prettify it"
+msgstr "Analizza e abbellisci il codice JSON"
+
+msgid "Minify JSON"
+msgstr "Comprimi JSON"
+
+msgid "Parse JSON and minify it"
+msgstr "Analizza e comprimi il codice JSON"
+
+msgid "Escape String"
+msgstr "Aggiungi sequenze di escape alle stringhe"
+
+msgid "Replace special characters with escape sequences"
+msgstr "Sostituisci i caratteri speciali con sequenze di escape"
+
+msgid "Escape HTML"
+msgstr "Aggiungi sequenze di escape a codice HTML"
+
+msgid "Replace special characters with HTML-safe sequences"
+msgstr "Sostituisci i caratteri speciali con sequenze HTML protette"
+
+msgid "Escape URL"
+msgstr "Aggiungi sequenze di escape a URL"
+
+msgid "Replace special characters with URL-safe sequences"
+msgstr "Sostituisci i caratteri speciali con sequenze protette per gli URL"
+
+msgid "Unescape String"
+msgstr "Rimuovi sequenze di escape"
+
+msgid "Replace escape sequences with characters"
+msgstr "Sostituisci le sequenze di escape con caratteri"
+
+msgid "Unescape HTML"
+msgstr "Rimuovi sequenze di escape da codice HTML"
+
+msgid "Replace HTML sequences with characters"
+msgstr "Sostituisci le sequenze HTML protette con caratteri"
+
+msgid "Unescape URL"
+msgstr "Rimuovi sequenze di escape da URL"
+
+msgid "Replace URL sequences with characters"
+msgstr "Sostituisci le sequenze protette per gli URL con caratteri"
+
+msgid "Convert JSON to YAML"
+msgstr "Converti JSON in YAML"
+
+msgid "Parse JSON and convert it to YAML document"
+msgstr "Analizza JSON e converti in documento YAML"
+
+msgid "Convert YAML to JSON"
+msgstr "Converti YAML in JSON"
+
+msgid "Parse YAML document and conver it to JSON"
```

Comment:
This typo has been fixed, it needs to be `"Parse YAML document and convert it to JSON"` now